### DISCUSSION TITLE
Concatenate HTTP responses & output to other Nodes

### DISCUSSION THREAD

**Discussion by MSpaceDev** [postmanlabs/postman-flows#365] [Ideas]
We are hoping to utilize flows in a way where we can run our **Cloud API-based test-suites** directly from Postman. We require the ability to concatenate 2 responses together, and output that to another Send Request node, or another concatenate node, etc.

The end goal would be to concatenate all our Failure & Success JSON responses together, as a string separated on `\n`, and then send that batched response to another endpoint via Send Request node, that reads all the data & outputs it to a live Data Studio on GCP.

It was quite disappointing to see that we could not take responses from Send Request nodes, and output them to other Send Request nodes.

Here is an example of our test-suite JSON output for one micro-service.
```json
{
    "Failures": null,
    "Successes": [
        "tests/test_all.gg->{Evaluate(\"tests/test_if_for_printf_set.gg\")}",
        "tests/test_all.gg->{Evaluate(\"tests/report/test_report.gg\")}",
        "tests/test_all.gg->{Evaluate(\"tests/batchlineincrement/test_batch_line_increment.gg\")}",
        "tests/test_all.gg->{Evaluate(\"tests/multitenancy/test_multi_tenancy.gg\")}",
        "tests/test_all.gg->{Evaluate(\"tests/waitread/test_wait_read.gg\")}",
        "tests/test_all.gg->{Evaluate(\"tests/test_cloud_upload_download.gg\")}",
        "tests/test_all.gg->{Evaluate(\"tests/code/test_code_main.gg\")}",
        "tests/test_all.gg: passed"
    ]
}
```

This is the best we can do right now, however the terminal output is not sufficient, as you can't link each response to their respective Node. _(The name of the node that provides the response is not outputted in Terminal)_.
There is also no opportunity for automation of checking batched results.
![Screenshot 2022-08-06 131700](https://user-images.githubusercontent.com/21283721/183246565-19d002a2-e7a5-4484-af47-be7a57cc0efd.png)

**Comment by ghost** [comment]
Maybe someone more knowledgeable will come around and point both of us in the right direction, but the way I've been concatenating results is via the "Create Data" block. Unfortunately, the "Create Data" block only allows you to concatenate two incoming flows at a time, so to concatenate more than 2 flows together at a time, I've been chaining them up:

![image](https://user-images.githubusercontent.com/79308268/183601676-4878fd70-f656-440b-909e-7042a03df1bc.png)

**Comment by Ariemeth** [comment]
I have been using conditionals to split the output from failures/errors and success like this
![image](https://user-images.githubusercontent.com/541023/183971757-193200f5-3a54-4a8b-a920-b388b28cbb33.png)
